etc/kayobe/ansible/octavia-amphora-image-register.yml

```diff
@@ -80,7 +80,7 @@
     # FIXME: Use 'tags' parameter of os_image module available from
     # openstack.cloud.image 1.5.0.
     - name: Ensure Octavia Amphora image is tagged
-      shell:
+      ansible.builtin.command:
```

Comment:
Should this be `ansible.builtin.comand`? You'd replaced `shell` with `ansible.builtin.shell` elsewhere.

Reply:
See https://ansible.readthedocs.io/projects/lint/rules/command-instead-of-shell/#command-instead-of-shell